sshd/zenith/sshd/tunnel.py

```diff
@@ -373,36 +380,33 @@ def consul_heartbeat(
         elif liveness_failures > 0:
             # We want services to stay in the critical state until they succeed at least once
             status = "warning" if liveness_succeeded_once else "critical"
-    print(f"[SERVER] [INFO] Updating service health status to '{status}' in Consul")
+    logger.debug(f"Updating Consul service health to '{status}'")
     # Post the service information to consul
     response = requests.put(
         f"{server_config.consul_url}/v1/agent/check/update/{tunnel.id}",
         json = { "Status": status }
     )
     if 200 <= response.status_code < 300:
-        print("[SERVER] [INFO] Service health updated successfully")
+        logger.info(f"Updated Consul service health to '{status}'")
         # Reset the Consul failures on success
         return 0, liveness_failures, liveness_succeeded_once
     else:
         # If we failed to update the health status, emit a warning
-        print("[SERVER] [WARNING] Failed to update service health", file = sys.stderr)
+        logger.warning("Failed to update Consul service health")
```

Comment:
It feels like we might need to just raise an exception here, and die?

Reply:
It is less disruptive in the case of, say, a Consul pod moving to another node, to just allow it to fail a couple of time, since it is just a liveness check. It will eventually cause a critical error and exit with an error if it persists.

Reply:
it feels like the bugs we have are some where in this retry logic, hence my thinking about failing hard on any errors here, so we can track them down.